### PR TITLE
Update advanced-concepts.md

### PR DESCRIPTION
**Pull request by DoviMaj**
- [x] You have read [The Odin Projects Contributing Guide](https://github.com/TheOdinProject/curriculum/blob/master/CONTRIBUTING.md).

#### 1.Describe the changes made:

- added links to resources
- fixed duplicate statement

#### 2. If this PR is related to an open issue please reference it with the `#` operator and the issue number below:

#XXXXX

### CODE REVIEW

File: javascript/react-js/advanced-concepts.md [hunk 1]
```diff
@@ -15,24 +15,24 @@ Good Luck!
...
+One common discussion about JavaScript is whether it would be better if you could declare types for variables and properties. Many programmers agree that the pattern of declaring types, which you have in many other programming languages is preferable because it allows you to catch errors, such as passing a string to a variable that should be a number. Therefore, React provides the possibility to declare types using [PropTypes](https://reactjs.org/docs/typechecking-with-proptypes.html). And if that's not enough for you, you could also use [TypeScript](https://www.typescriptlang.org/) with React, here is an [example](https://youtu.be/Z5iWr6Srsj8) of using typescript with React.
```
Comment (I3uckwheat) [inline javascript/react-js/advanced-concepts.md:18]: I do not think we want to muddy the water with typescript at this stage. This should be removed entirely IMO

**Review by I3uckwheat — CHANGES_REQUESTED** [review]
Other than Typescript, everything else here is great, thank you!

File: javascript/react-js/advanced-concepts.md (hunk 1 above)
Comment (marvingay) [reply javascript/react-js/advanced-concepts.md:18]: It was only left in tact, more as a "To-do" if we ever wanted to expand lessons for any of these concepts down the line.

Comment (DoviMaj) [reply javascript/react-js/advanced-concepts.md:18]: So should i leave it or remove it? 

Or just remove the video reference for typescript?

Comment (marvingay) [reply javascript/react-js/advanced-concepts.md:18]: Remove the video reference. Per discussion among the maintainers, the documentation is the most appropriate entry point for this extra topic.

Comment (DoviMaj) [reply javascript/react-js/advanced-concepts.md]: done

**Review by I3uckwheat — APPROVED** [review]

**Pull request merged**